build.gradle

```diff
@@ -246,6 +246,7 @@ task fastbuild(dependsOn: [
 
 task travisbuild(dependsOn: [
     fastbuild,
+    ':dbfit-java:db2:integrationTest',
```

Comment:
It's more about how long it takes for a failing test to fail. But anyway - I guess the difference is negligible compared to the total time with the setup before that. OK to be as is.